habitat_baselines/common/env_utils.py

```diff
@@ -0,0 +1,141 @@
+import random
+
+import numpy as np
+
+import habitat
+from habitat import Config, SimulatorActions, make_dataset
+
+
+class NavRLEnv(habitat.RLEnv):
+    def __init__(self, config_env, config_baseline, dataset):
+        self._config_env = config_env.TASK
+        self._config_baseline = config_baseline
+        self._previous_target_distance = None
+        self._previous_action = None
+        self._episode_distance_covered = None
+        super().__init__(config_env, dataset)
+
+    def reset(self):
+        self._previous_action = None
+
+        observations = super().reset()
+
+        self._previous_target_distance = self.habitat_env.current_episode.info[
+            "geodesic_distance"
+        ]
+        return observations
+
+    def step(self, action):
+        self._previous_action = action
+        return super().step(action)
+
+    def get_reward_range(self):
+        return (
+            self._config_baseline.BASELINE.RL.SLACK_REWARD - 1.0,
```

Comment:
Same here. These should be `MODEL` instead of `BASELINE`. I might be missing some context though.

Reply:
So, if I have to use my own model, I will have to create a config with `BASELINE` attribute in it?

Reply:
If someone implements another algorithm, say A2C, they will create algorithm-specific config under `BASELINE.RL.A2C`. In this particular case tho, `SLACK_REWARD` is not algorithm-specific and is more related to the task and environment, so it lives under `BASELINE.RL`. Then again, it's probably a good idea to have a more informative/representative name than **BASELINE**